### PR TITLE
feat: Adding introduction content

### PR DESCRIPTION
**Pull request by mbonig**
---
By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

File: docusaurus/docs/introduction/the-projen-workflow.md [hunk 1]
```diff
@@ -2,4 +2,44 @@
 sidebar_position: 2
 ---
 
-# The Projen Workflow
\ No newline at end of file
+# The Projen Workflow
+
+The projen workflow is very simple:
+
+1. Initialize a project type using the cli (`npx projen`).
```
Comment (mikejgray) [inline docusaurus/docs/introduction/the-projen-workflow.md:9]:
```suggestion
1. Initialize a project type using the cli (`npx projen new`).
```

**Review by mikejgray — CHANGES_REQUESTED** [review]
While I was writing my docs, I was thinking we should probably standardize on how we reference projen's name. The initial page anyone sees in the doc uses italics (_projen_). I like the idea of styling it every time it's used, but I'm equally open to having no styling. Thoughts?

**Comment by mbonig** [comment]
> While I was writing my docs, I was thinking we should probably standardize on how we reference projen's name. The initial page anyone sees in the doc uses italics (_projen_). I like the idea of styling it every time it's used, but I'm equally open to having no styling. Thoughts?

I'm fine with no styling. I think it'd be too easy to miss one somewhere.

File: docusaurus/docs/introduction/the-projen-workflow.md [hunk 2]
```diff
@@ -2,4 +2,44 @@
...
+
+Running `npx projen new` will initialize the current directory with the new project type specified.
+If the directory is empty, then the files under control by the project type will be created and
+the git repository will be initialized with a baseline commit.
+
+:::warning
+projen allows you to "take over" an existing project. 
+A directory does NOT need to be empty to run the `npx projen new` command and start a new project.
+However, it will overwrite any existing files that are managed by projen. 
+Make sure you have all your changes committed before running `npx projen new`.
```
[the remark this answers is not in the document]
Comment (mikejgray) [reply docusaurus/docs/introduction/the-projen-workflow.md:23]: I was thinking more along the lines of warning newcomers that you're going to have a bunch of files added/updated/replaced, and where to find out exactly what changed. Even though I've used it frequently, it's still a little jarring to me when suddenly a bunch of new stuff appears and I don't see it show up in my git changes.

[the remark this answers is not in the document]
Comment (mbonig) [reply docusaurus/docs/introduction/the-projen-workflow.md:23]: Ok, I took another stab at it but if that's still not getting across the point I'm going to need some help with the wording.

**Review by mikejgray — APPROVED** [review]

**Pull request merged**